interface.go

```diff
@@ -352,15 +366,14 @@ type RetryableQueue interface {
 	// and a bool indicating whether the job was found or not.
 	GetAttempt(ctx context.Context, id string, attempt int) (RetryableJob, bool)
 
-	// CompleteAndPut marks an existing job toComplete in the queue (see
-	// CompleteRetry) as finished processing its retry and inserts a new job
-	// toPut in the queue (see Put). Implementations must make this operation
-	// atomic.
-	CompleteAndPut(ctx context.Context, toComplete, toPut Job) error
+	// CompleteRetryingAndPut marks an existing job toComplete in the queue (see
+	// CompleteRetrying) as finished retrying and inserts a new job toPut in the
+	// queue (see Put). Implementations must make this operation atomic.
+	CompleteRetryingAndPut(ctx context.Context, toComplete, toPut RetryableJob) error
 
-	// CompleteRetry marks a job that needs to retry as finished processing, so
+	// CompleteRetrying marks a job that is retrying as finished processing, so
 	// that it will no longer retry.
-	CompleteRetry(ctx context.Context, j RetryableJob) error
+	CompleteRetrying(ctx context.Context, j RetryableJob) error
```

Comment:
I thought `CompleteRetry` was clear and bit more concise, but I am not too opinionated on this change.

Reply:
I mostly did this to be clearer and consistent with the naming of each state. In the [other PR](https://github.com/mongodb/amboy/pull/192), I referred to jobs that are currently in the process of retrying as "retrying" to distinguish this status from just regular "retryable" (but not actively retrying) jobs. I think I would prefer to keep it like this to reduce ambiguity about what state transition it's used for. Without specifying that the job is retrying, it's unclear whether it means "complete executing a retryable job" (not what it means, retryable jobs still call `Complete()` like normal jobs) versus "marking a job as no longer retrying".